src/software/jetson_nano/services/power.cpp

```diff
@@ -1,12 +1,17 @@
 #include "software/jetson_nano/services/power.h"
 
 #include <boost/bind/bind.hpp>
+#include <boost/filesystem.hpp>
 #include <cstdint>
 
 #include "proto/power_frame_msg.nanopb.h"
 
 PowerService::PowerService()
 {
+    if (!boost::filesystem::exists(DEVICE_SERIAL_PORT))
+    {
+        LOG(FATAL) << "PLUG THE USB INTO THE JETSON NANO";
+    }
```

Comment:
will we not have comm over UART through the jetson pin headers?

Reply:
this will get removed when we switch to UART instead of USB